src/document.rs

```diff
@@ -194,6 +196,7 @@ impl Document {
                  -> Result<Document> {
         trace!("Parsing {:?}", rel_path);
         let content = files::read_file(src_path)?;
+        let content = String::from_iter(normalized(content.chars()));
```

Comment:
I'm sorry I didn't notice this before but ... what do you think of doing the normalization in `read_file`?

I could see it being nice that `read_file` is a general abstraction for loading data throughout cobalt.  If there are cases where we shouldn't normalize (like maybe `cobalt migrate`), then we just don't use it.

Reply:
… I didn't notice read_file was ours… ^^'